apps/api-gateway/src/services/report_export_service.py

```
@@ -203,6 +211,190 @@ async def _get_transactions(
+            data = await self._get_transactions(start_date, end_date, store_id, db)
+            ws.title = "交易明细"
+            self._write_transactions_xlsx(ws, data, header_font, header_fill, right_align)
+        else:
+            raise ValueError(f"不支持的报表类型: {report_type}")
+
+        output = io.BytesIO()
+        wb.save(output)
+        return output.getvalue()
+
+    def _write_income_statement_xlsx(self, ws, data, start_date, end_date,
+                                      header_font, header_fill, section_font, section_fill,
```

Ruff (0.15.2)

[warning] Unused method argument: `header_font`

(ARG002)